samples/bluetooth/broadcast_audio_sink/Kconfig

```
@@ -28,4 +28,11 @@ config SCAN_DELAY
 	  Time to advertise connectable for a Broadcast Assistant to connect
 	  before starting scanning for Broadcast Sources.
 
+config BT_TARGET_BROADCAST_NAME
```

```suggestion
config TARGET_BROADCAST_NAME
```

Should not use the `BT_` prefix for samples